### ISSUE TITLE
EN 15804+A2 chapter 7.3.4, EoL Scenario information

### ISSUE THREAD

**Issue by RobertSpang** [InDataWG/ILCD-EPD-Data-Format#16]
### I raise this change request because...

- [ ] the ILCD+EPD format lacks compliance with regualtions.
- [X] the ILCD+EPD format lacks conformaty with standards.
- [ ] the ILCD+EPD format needs to be improved regarding existing format or master data elements.
- [ ] there is another problem or use case I´m trying to address.

### Problem description

![EN 15804+A2 chapter 7 3 4_EoL](https://github.com/InDataWG/ILCD-EPD-Data-Format/assets/150464594/35359863-b275-45f9-85eb-19b3996ac134)

### Solution description

New master datasets need to be created and possibly the format extended

### Relevance or added value

Medium relevance

**Comment by RobertSpang** [comment]
Our recommendation is to just provide the amounts of the unit column.
Two sections would make sense:

#1
kg collected separately
kg collected with mixed construction waste
--> needs to sum up to the declared unit

#2
kg for re-use
kg for recycling
kg for energy recovery
kg product or material for final deposition
--> needs to sum up to the declared unit

The assumptions for scenatrio development are free text and are covered by existing fields already.

**Comment by okworx** [comment]
implement as suggested

**Comment by RobertSpang** [comment]
yes. for validation, minor deviations regarding mass shall be accepted (e.g. due to rounding)
